apps/TideForker/main.cpp

```diff
@@ -66,6 +66,7 @@ int main(int argc, char* argv[])
         MPIChannelPtr worldChannel(new MPIChannel(argc, argv));
         const int rank = worldChannel->getRank();
         MPIChannelPtr localChannel(new MPIChannel(*worldChannel, 0, rank));
+        MPIChannelPtr localChannel2(new MPIChannel(*worldChannel, 0, rank));
```

Comment:
give that a better man to declare what this is used for?

Reply:
agree, this has to be renamed.